_includes/tools/ava/usage.md

```diff
@@ -0,0 +1,40 @@
+AVA comes with built-in support for Babel 6 and let you write your tests
```

Comment:
hi @hzoo I basically ripped off the text from https://github.com/sindresorhus/ava#es2015-support almost with no changes :smile:

something like this:
> AVA ships with ES2015 support for test files built-in using Babel 6 under the hood.
>
> The AVA default configuration includes the `"es2015"`and `"stage-0"` Babel 6 presets, but you can customize any Babel option for transpiling test files with the `babel` option in AVA's [`package.json` configuration](https://github.com/sindresorhus/ava#configuration).

will work?

I'm not a native speaker so please bare with me :smiley:

Thanks!